pkg/services/ldap/ldap.go

```diff
@@ -427,6 +427,12 @@ func (server *Server) buildGrafanaUser(user *ldap.Entry) (*models.ExternalUserIn
 		}
 	}
 
+	// If there are groups org mappings configured but no matching mappings
```

Comment:
I think "group org mappings" is correct?
```suggestion
	// If there are group org mappings configured, but no matching mappings,
```